src/services/replyComments.js

```diff
@@ -12,24 +15,43 @@ class ReplyCommentsService {
 
     if (!checkComment) return res.boom.notFound('Comment not found.');
 
-    const comment = await new ReplyComments(req.body).save();
+    req.body.userId = req.session.passport.user.id;
+    const replyComment = await new ReplyComments(req.body).save();
 
-    if (comment) {
+    if (replyComment) {
       await Comments.updateOne(
         { _id: req.body.commentId },
-        { $push: { replyComments: comment._id } }
+        { $push: { replyComments: replyComment._id } }
       ).exec();
     }
 
-    return comment ? res.json(true) : res.json(false);
+    return replyComment ? res.json(true) : res.json(false);
   }
 
   async updateReplyComment (req, res) {
-    const comment = await ReplyComments.findByIdAndUpdate(req.params.id, {
-      comment: req.body.comment
-    }).exec();
+    const replyComment = await ReplyComments.findById(req.params.id).exec();
+
+    if (!replyComment) return res.boom.notFound();
+
+    const checkAuth = await commentsService.checkAuth(
```

Comment:
You are still using verbs for names... Use some proper names. For example: "isAdmin"